src/analysis/calculus/conformal.lean

```diff
@@ -0,0 +1,156 @@
+/-
+Copyright (c) 2021 Yourong Zang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yourong Zang
+-/
+import analysis.normed_space.conformal_linear_map
+
+/-!
+# Conformal Maps
+
+A continuous linear map between real normed spaces `X` and `Y` is `conformal_at` some point `x`
+if it is real differentiable at that point and its differential `is_conformal_linear_map`.
+
+## Main results
+
+* `conformal_at`: the main definition
```

Comment:
Here and in the other module docstring -- `conformal_at` and `is_conformal_map` are definitions, not results; you can either put them under a new "main definitions" section, or you can describe them in the introductory paragraph.  `conformal_factor_at` is also a definition, not a result.

Reply:
Thank you for letting me know. I will create a new section.